friendsofsymfony/user-bundle/2.0/config/packages/fos_user.yaml

```diff
@@ -0,0 +1,8 @@
+# Read the documentation: https://symfony.com/doc/master/bundles/FOSUserBundle/index.html
+fos_user:
+#    db_driver: orm # other valid values are 'mongodb' and 'couchdb'
+    firewall_name: main
```

Comment:
Instead, it's better IMO to directly uncomment `db_driver`, `firewall_name` and `user_class` in the recipe as they are all mandatory anyway in the bundle.

Reply:
Thanks, you're right. I changed the config in e129d94